packages/openactive-integration-tests/test/templates/b-req.js

```diff
@@ -253,6 +263,7 @@ const bReqTemplates = {
   noBrokerName: createNoBrokerNameBReq,
   incorrectTotalPaymentDuePrice: createIncorrectTotalPaymentDuePriceBReq,
   incorrectOrderDueToMissingPaymentProperty: createIncorrectOrderDueToMissingPaymentProperty,
+  incorrectOrderDueToUnnecessaryPaymentProperty: createIncorrectOrderDueToUnnecessaryPaymentProperty,
   incorrectOrderDueToMissingIdentifierInPaymentProperty: createIncorrectOrderDueToMissingIdentifierInPaymentProperty,
```

Comment:
I notice these are named with the prefix `incorrect`... should they be? For `incorrectTotalPaymentDuePrice` the "incorrect" refers to the price, which is intentionally incorrect. However for the three lines above "incorrect" really depends on context... a missing payment property is correct behaviour in some circumstances (e.g. when prepayment is unavailable)

Reply:
@nickevansuk Totally agreed. I renamed the above one to `noPayment`. It's just missing a payment property. Whether that's incorrect or not depends on the test